ducktape/cluster/remoteaccount.py

```diff
@@ -62,6 +63,8 @@ def _can_ping_url(self, url, headers):
     def ssh_command(self, cmd):
         if self.local:
             return cmd
+        if self.sudo_user is not None:
```

Comment:
@granders Today if I want to kill a process on a remote machine as root user, I won't be able to do that with either `RemoteAccount.signal(...)` or `RemoteAccount.kill(...)`. And if I want to start a process as root user, I have do explicitly do `ssh_output("sudo -u root " + cmd)`, which is not as nice as being able to do `ssh_output(cmd, sudo_user=True)`. Does this make sense?

I agree with you that the second format, i.e. `node.account.ssh(cmd, sudo=True)`, is better.